### DISCUSSION TITLE
I wonder that If i know user wallet and user address, Can i skip linker process

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#194] [Q&A] [answered]
As title say that, Can i possible??

Just i want to skip connectData.Approval process.

So i immediately use sendTraction method.

I consider that how to avoid infinity connecting screen.

Always thank you~

**Comment by skibitsky** [comment, marked as answer]
Hello,

You have to go through approval process to establish secure connection.

I made some changes in the latest release that can improve the connection with OKX. Could you upgrade and try again, please?

**Reply by matthew2099** [reply]
Oh thank you.
I try that!

**Reply by matthew2099** [reply]
@skibitsky 
I test that.
I think message work well.
Just sometimes I see "Network Error and Try again". But connect is very well. Still i don't understand
Anyway thank you skibitsky.
I think I decided it was enough to use.

**Reply by skibitsky** [reply]
The "Network Error and Try again" happens due to the low timeout on the OKX Wallet side. I'll ask them to extend it.
